storage/innobase/log/log0log.cc

```diff
@@ -1000,9 +1083,7 @@ void log_write_up_to(lsn_t lsn, bool durable,
   }
 
   if (durable)
-  {
```

Comment:
According to style guide,  curly brackets around single statement are OK.
This is also unrelated to the goal of this patch.
Can we leave it as-is, please, for now?